src/mapping.cpp

```diff
@@ -0,0 +1,181 @@
+#include "mapping.hpp"
+
+#include <string>
+
+const std::string keystonesJson = R"(
```

Comment:
I moved the declaration of these constants to mapping.cpp, so that mapping.hpp doesn't bring in the full text of all of these into every compilation unit that includes mapping.hpp due to needing to refer to any of these strings. Since the strings are spread out across a few files, this should marginally improve compilation time, especially incremental compilation of those files. See mapping.hpp declares these with "extern" meaning they're defined in some compilation unit (mapping.o in this case) and the linker will sort it out, so the compiler can make progress without the definitions.

Reply:
I didnt actually think about this but that could work as well. I also didnt know about extern. Thanks for the tip! Ill drill down into the C++ documentation to get a general better handle on extern.